project1/import.py

```diff
@@ -0,0 +1,38 @@
+import os
+
+from flask import Flask, session
+from flask_session import Session
+from sqlalchemy import create_engine
+from sqlalchemy.orm import scoped_session, sessionmaker
+from flask import render_template,  request, session
+from flask_session import Session
+from books import *
+import csv
+
+app1 = Flask(__name__)
+
+
+# Configure session to use filesystem
+app1.config["SESSION_PERMANENT"] = False
```

Comment:
this part is redundant. make sure you remove from application.py

Reply:
I have removed the unnecessary and redundant lines from the imports column from application.py and import.py.  I have made 2 commits for changes made in both files.